grafana/scylla-cql.2019.1.template.json

```diff
@@ -374,6 +374,159 @@
                 ],
                 "title": "New row"
             },
+            {
+                "class": "row",
+                "panels": [
+                    {
+                        "class": "gauge_errors_panel",
+                        "description": "Using consistency level ANY in a query will hurt persistency, if the node receiving the request will fail the data may be lost",
+                        "targets": [
+                            {
+                                "expr": "floor(100 *sum(delta(scylla_query_processor_queries{instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\", consistency_level=\"ANY\"}[30s]))/sum(delta(scylla_query_processor_queries{instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\"}[30s]))) OR vector(0)",
+                                "format": "time_series",
+                                "hide": false,
+                                "instant": false,
+                                "interval": "",
+                                "intervalFactor": 1,
+                                "legendFormat": "aaa",
+                                "refId": "A"
+                            }
+                        ],
+                        "title": "CQL ANY Queries"
+                    },
+                    {
+                        "class": "ops_panel",
+                        "description": "Using consistency level ANY in a query will hurt persistency, if the node receiving the request will fail the data may be lost",
+                        "span": 3,
+                        "targets": [
+                            {
+                                "expr": "sum(delta(scylla_query_processor_queries{instance=~\"[[node]]\",cluster=~\"$cluster|$^\", dc=~\"$dc\", shard=~\"[[shard]]\",  consistency_level=\"ANY\"}[1m])) by ([[by]])",
+                                "format": "time_series",
+                                "hide": false,
+                                "intervalFactor": 2,
+                                "refId": "A"
+                            }
+                        ],
+                        "title": "CQL ANY CL Queries"
+                    },
+                    {
+                        "class": "gauge_errors_panel",
+                        "description": "Using consistency level ALL in a query will hurt availability, if the node is unavailable operations will fail",
```

Comment:
"if the node is unavailable operations will fail" --> "if a node is unavailable operations will fail",